.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  binaries:
+    strategy:
+      matrix:
+        goos: [linux]
+        goarch: [amd64, arm64, ppc64le, s390x]
```

Comment:
Would it perhaps be proactive to create a build tags dimension here with a single entry for now? Or, would it be simpler to have the vanilla pebble without any additional information:

For example:

`ARCHIVE_FILE=pebble_${GITHUB_REF_NAME}_${VARIANT}_${GOOS}_${GOARCH}.tar.gz`

**# Case always including a variant:**

```
pebble_v1.0_default_linux_amd64.tar.gz
pebble_v1.0_termus_linux_amd64.tar.gz
:
```

or simply:

```
pebble_v1.0_linux_amd64.tar.gz
pebble_v1.0_termus_linux_amd64.tar.gz
:
```

In the former case the file names have the same information at the same delimited offsets.

Reply:
I'd prefer we keep the names simple for the common case (just using goos_goarch is very common for naming Go binaries), and add the variant to the end as needed. I'd prefer it was on the end anyway, like `pebble_v1.0.0_linux_amd64_termus.tar.gz`.